src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-color;
+  border-radius: $border-radius;
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    margin: 0 0 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-accent-color;
+    margin: 0 0 4px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-color;
+    margin: 0 0 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-total {
+    display: flex;
+    margin: 0;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
Add default `font-size` and `font-weight` to your `html`, not to duplicate it in every class